src/bundle/Resources/views/content/widgets/content_edit.html.twig

```diff
@@ -8,10 +8,6 @@
             'hidden': 'hidden',
             'class': 'ez-extra-actions__form-field'
         } }) }}
-        {{ form_widget(form.version_info, { 'attr': {
```

Comment:
How removing this field would affect version/draft conflict modals? are those not used by js files?